### DISCUSSION TITLE
How to highlight underscore separated words similar to diff-so-fancy

### DISCUSSION THREAD

**Discussion by abhijithcd** [dandavison/delta#1575] [Q&A]
![delta](https://github.com/dandavison/delta/assets/7361708/ccbb06aa-9320-4fee-8fad-172218debb3f)
Delta

![diff_so_fancy](https://github.com/dandavison/delta/assets/7361708/7741594d-13fa-41d2-a1b6-e62e6cc6c3fe)
diff-so-fancy

diff-so-fancy emphasizes underscore separated words as shown above. I tried the emulation mode as well, but it didn't show as expected.
Is there way to achieve that in delta?

**Comment by dandavison** [comment]
Hi @abhijithcd, have you experimented with `word-diff-regex`?

**Reply by abhijithcd** [reply]
In another case, diff-so-fancy highlighted everything between the two changes within a line even though there were only 2 changes. 
I found this one is more granular but looks better in most cases. `--word-diff-regex='[^[:space:]]'`
